pkg/database/database.go

```diff
@@ -40,15 +42,25 @@ func NewClient(config *csconfig.DatabaseCfg) (*Client, error) {
 	return &Client{Ent: client, CTX: context.Background()}, nil
 }
 
-func (c *Client) IsMachineRegister(machineID string) (bool, error) {
-	exist, err := c.Ent.Machine.Query().Where().Select(machine.FieldMachineId).Strings(c.CTX)
-	if err != nil {
-		return false, err
-	}
-	if len(exist) > 0 {
-		return true, nil
+func (c *Client) StartFlushScheduler(config *csconfig.FlushDBCfg) (*gocron.Scheduler, error) {
+	maxAge := time.Duration(0)
+	maxItems := 0
+	if config.MaxItems != nil && *config.MaxItems <= 0 {
+		return nil, fmt.Errorf("max_items can't be negatif number")
 	}
 
-	return false, nil
+	maxItems = *config.MaxItems
+	if config.MaxItems != nil && *config.MaxAge != "" {
+		ageDuration, err := time.ParseDuration(*config.MaxAge)
```

Comment:
For users simplicity, don't we want to automatically assume that MaxAge is in days ? Go's duration format doesn't go further than hours, so forcing the user to convert days / months into hours can be annoying ?